apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SimpleSubscriberTest.cs

```diff
@@ -175,18 +183,21 @@ public override Task WriteCompleteAsync()
                 }
             }
 
-            public FakeSubscriber(IEnumerator<IEnumerable<ServerAction>> msgsEn, IScheduler scheduler, IClock clock, TaskHelper taskHelper)
+            public FakeSubscriber(
```

Comment:
What is a "write async predelay"?

Reply:
Simulation of network or server slowness. Comment added.